float32 x_trans 
float32 y_trans
float32 z_trans
float32 x_rot #first element of the quaternion
float32 y_rot #second element of the quaternion
float32 z_rot #third element of the quaternion
float32 w #forth element of the quaternion
string segment_name #name of a specific component of the object
string subject_name #name of the entire object
int32 frame_number #unit of time for each capture
string translation_type #Local or Global
